components/layout/components/github-edit.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import { GithubOutlined } from '@ant-design/icons';
+
+const propTypes = {
+  query: PropTypes.objectOf(PropTypes.object()).isRequired,
+};
+
+const GithubEdit = ({ query }) => {
+  const [cat, slug] = query['documentation-page'];
```

Comment:
For future reference please always use descriptive names for variables and methods. That way it is easy for everyone to understand what they mean? i.e. `cat` = `category`.

```suggestion
  const [category, page] = query['documentation-page'];
```